docs/reference/components/sinks/redis.cue

```diff
@@ -0,0 +1,127 @@
+package metadata
+
+components: sinks: redis: {
+	title: "Redis"
+	classes: {
+		commonly_used: false
+		delivery:      "best_effort"
+		development:   "beta"
+		egress_method: "stream"
+		service_providers: []
+		stateful: false
+	}
+	features: {
+		buffer: enabled:      false
```

Comment:
```suggestion
		buffer: enabled:      true
```

I'm not even sure why this can be disabled, the buffers are implemented at the topology level.